tests/dags/providers/provider_api_scripts/resources/inaturalist/pull_sample_records.py

```diff
@@ -0,0 +1,166 @@
+"""
+This is a rarely used thing for generating sample files and moving them in and out of
+the github project.
+
+This is used exclusively for local development and is not intended to be run
+as part of a production workflow.
```

Comment:
Is there a way to prevent it from being accidentally run outside of a local environment?

Reply:
I mean, yeah... I guess there's a continuum from it doesn't belong in the repo at all; to put it in, but comment everything out; to it's in the test resource directory and it doesn't have "test" in the file name and it refers to directories that don't exist, so even if it did get accidentally run, nothing would happen... you think I should comment everything out? Keep it out entirely?

Reply:
Maybe if there's an environment variable we can check that automatically prevents it from running outside of a local environment, that would be sufficient for me personally. I think it's fine to keep if it's useful for testing. Alternatively, maybe it's better to throw in a gist or wiki page or something and link it from the DAGs docs?

Reply:
Ooh! Yeah, a gist or wiki makes more sense to me, since it's really intended as "hey, this is how I did this, in case you need to set up testing with another dataset". Is there an example gist I could consult for style and so forth?

Reply:
I don't have an example, but I think it would potentially fit into the https://make.wordpress.org/openverse/handbook/.

@zackkrida do you know what the process is to get Rebecca able to add a page to the handbook?

Alternatively, one of us could create it for you and put the script in there.

If we had directories per provider DAG then it'd be nice to have additional documentation put into an accompanying README.

Reply:
FWIW the `tests/` folder does not get loaded into the container at runtime, only `openverse_catalog/dags` does:
https://github.com/WordPress/openverse-catalog/blob/06046dae2e5ae2f0d574073c0ce4199295249796/docker-compose.yml#L13

So in non-dev environments, this file would not be available to run within the container. Definitely not against moving this to the handbook or elsewhere but we thankfully don't need to worry about the risk of running this in production 🙂

Reply:
Ah perfect!

Reply:
And, I remembered your trick about the prod environment not having pytest installed, so I added that too.

Reply:
Yea that bit I did in the other PR was because we needed to reference `pytest_socket`, I think removing it was the right call!